trace/tests/unit/ext/flask/app/main.py

```diff
@@ -58,5 +59,29 @@ def query():
         return "Query failed. Check your env vars for connection settings."
 
 
+@app.route('/postgresql')
+def postgresql_query():
+    try:
+        conn = psycopg2.connect(
+            host='localhost',
+            user='postgres',
```

Comment:
We should probably have a single file shared between the flask and django tests with the username, password, dbname, hostname, and sample queries and expected results.

Reply:
Done.